src/sentry/api/endpoints/codeowners/external_actor/team_details.py

```diff
@@ -1,81 +1,5 @@
-import logging
-from typing import Any
+from sentry.integrations.api.endpoints.codeowners.external_actor.team_details import (
```

Comment:
Do we need all these directories? We could flatten this to `sentry.integrations.api.endpoints.codeowners` and rename endpoint module to `external_actor_team_details`.

Reply:
👍 that sounds cleaner